### PR TITLE
PROD-1899 - Enhancement Customize group directories

### PR DESCRIPTION
**Pull request by krunal-buddyboss**
### Jira Issue:
https://buddyboss.atlassian.net/browse/PROD-1899

### General Note
Keep all conversations related to this PR in the associated Jira issue(s). Do NOT add comment on this PR or edit this PR’s description.

### Notes to Developer
* Ensure the IDs (i.e. PROD-1) of all associated Jira issues are reference in this PR’s title
* Ensure that you have achieved the Definition of Done before submitting for review
* When this PR is ready for review, move the associate Jira issue(s) to “Needs Review” (or “Code Review” for Dev Tasks)

### Notes to Reviewer
* Ensure that the Definition of Done have been achieved before approving a PR
* When this PR is approved, move the associated Jira issue(s) to “Needs QA” (or “Approved” for Dev Tasks)

### CODE REVIEW

File: src/bp-core/admin/bp-core-admin-settings.php [hunk 1]
```diff
@@ -1885,6 +1883,78 @@ function bp_admin_setting_group_layout_default_option() {
...
+						'centered' => array(
+							'label' => 'Centered',
+							'class' => 'option opt-centered',
+						),
+					),
+				)
+			);
+		?>
+	</div>
+	<p class="description"><?php _e( 'Select the style of the of grid layouts. Group avatars and cover images will only be displayed if they are enabled.', 'buddyboss' ); ?></p>
```
Comment (chetansatasiya) [inline src/bp-core/admin/bp-core-admin-settings.php:1917]:
```suggestion
	<p class="description"><?php esc_html_e( 'Select the style of the of grid layouts. Group avatars and cover images will only be displayed if they are enabled.', 'buddyboss' ); ?></p>
```
```suggestion
	<p class="description"><?php _e( 'Select the style of the of grid layouts. Group avatars and cover images will only be displayed if they are enabled.', 'buddyboss' ); ?></p>
```

File: src/bp-core/admin/bp-core-admin-settings.php [hunk 2]
```diff
@@ -1885,6 +1883,78 @@ function bp_admin_setting_group_layout_default_option() {
...
+ *
+ * @uses checked() To display the checked attribute
+ */
+function bb_admin_setting_group_elements( $args ) {
+
+	echo "<div class='bb-group-elements'>";
+	foreach ( $args['elements'] as $element ) {
+		$element_name = $element['element_name'];
+		?>
+		<div class="bb-group-element bb-group-element-<?php echo $element_name; ?>">
```
Comment (chetansatasiya) [inline src/bp-core/admin/bp-core-admin-settings.php:1936]:
```suggestion
		<div class="bb-group-element bb-group-element-<?php echo esc_attr( $element_name ); ?>">
```
```suggestion
		<div class="bb-group-element bb-group-element-<?php echo $element_name; ?>">
```

File: src/bp-core/admin/bp-core-admin-settings.php [hunk 3]
```diff
@@ -1885,6 +1883,78 @@ function bp_admin_setting_group_layout_default_option() {
...
+					'selected' => $element_name,
+				)
+			);
+			?>
+		</div>
+		<?php
+	}
+	echo '</div>';
+	echo '<p class="description">';
+	echo _e( 'Select which elements show in group directories. Cover images will only display in grid view and group descriptions will only display in list view.', 'buddyboss' );
```
Comment (chetansatasiya) [inline src/bp-core/admin/bp-core-admin-settings.php:1954]:
```suggestion
	echo esc_html_e( 'Select which elements show in group directories. Cover images will only display in grid view and group descriptions will only display in list view.', 'buddyboss' );
```
```suggestion
	echo _e( 'Select which elements show in group directories. Cover images will only display in grid view and group descriptions will only display in list view.', 'buddyboss' );
```

File: src/bp-groups/bp-groups-functions.php [hunk 4]
```diff
@@ -4500,3 +4504,77 @@ function groups_can_user_manage_video_albums( $user_id, $group_id ) {
 
 	return $is_allowed;
 }
+
+function bb_groups_loop_members( $group_id = 0, $role = array( 'member', 'mod', 'admin' ) ) {
```
Comment (chetansatasiya) [inline src/bp-groups/bp-groups-functions.php:4508]: Need to add function doc block

File: src/bp-groups/bp-groups-functions.php [hunk 5]
```diff
@@ -4500,3 +4504,77 @@ function groups_can_user_manage_video_albums( $user_id, $group_id ) {
...
+					'avatar_dir' => 'avatars',
+					'object'     => 'user',
+					'type'       => 'thumb',
+					'html'       => false,
+				)
+			);
+			?>
+			<span class="bs-group-member" data-bp-tooltip-pos="up-left" data-bp-tooltip="<?php echo esc_attr( bp_core_get_user_displayname( $member->ID ) ); ?>">
+				<a href="<?php echo esc_url( bp_core_get_user_domain( $member->ID ) ); ?>">
+					<img src="<?php echo $avatar; ?>"
```
Comment (chetansatasiya) [inline src/bp-groups/bp-groups-functions.php:4550]:
```suggestion
					<img src="<?php echo esc_url( $avatar ); ?>"
```
```suggestion
					<img src="<?php echo $avatar; ?>"
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]

**Comment by krunal-buddyboss** [comment]
@chetansatasiya Applied the above-suggested changes.

**Pull request closed without merging**